### DISCUSSION TITLE
react-fontawesome stop working properly after upgrading from 9.5.5 to 10.0.3

### DISCUSSION THREAD

**Discussion by jfparis84** [vercel/next.js#19585] [Help]
After upgrading to the latest version of nextjs, all of the svg from font-awesome show up but the CSS doesn't being loaded in the 'head' section. 

In the 9.5.5, we can see the fontawesome stylesheet being injected in the 'head' section.
In the 10.0.3, the stylesheet is not injected in the 'head' section.

The stylesheet is important to apply some of the helper classes like 'fa-x6'
`<FontAwesomeIcon icon="spinner" size="6x" />`

### Nextjs 9.5.5
<img width="1200" alt="Capture d’écran, le 2020-11-26 à 22 40 28" src="https://user-images.githubusercontent.com/6884684/100408303-ac2f1280-3038-11eb-89bd-9ea81d17aa51.png">

### Nextjs 10.0.3
<img width="1203" alt="Capture d’écran, le 2020-11-26 à 22 38 44" src="https://user-images.githubusercontent.com/6884684/100408314-b4874d80-3038-11eb-9653-c645a8954486.png">

**Comment by GilesC9** [comment]
I had this issue, I found this thread which seemed to do the trick for me.

https://github.com/FortAwesome/react-fontawesome/issues/234

**Reply by jfparis84** [reply]
Thanks a lot, I will take a look :)
